app/params/config.go

```diff
@@ -20,6 +20,18 @@ const (
 	Bech32PrefixAccAddr = "bbn"
 )
 
+// taken from https://github.com/celestiaorg/celestia-app/pull/2985
+const (
+	// DefaultMinGasPrice is the default min gas price that gets set in the app.toml file.
+	// The min gas price acts as a filter. Transactions below that limit will not pass
+	// a nodes `CheckTx` and thus not be proposed by that node.
+	DefaultMinGasPrice = 0.002
```

Comment:
The gas price is already in microbbn `1ubbn` == 1^10-6 bbn
setting the minGasPrice as 1ubbn is too much?

Reply:
Imo 1ubbn is too much. Typically a tx costs >100,000 gas (sometimes it can be 500,000 - 1,000,000 ubbn). With 1ubbn gas price it means that a tx costs at least 0.1bbn. This is a bit too much for an appchain.

Probably that's why Celestia uses 0.002, which is the value here